examples/healthData/src/healthDataSchema.ts

```diff
@@ -0,0 +1,67 @@
+// The following is a schema definition for enetring health data.
+
+export interface HealthDataResponse {
+    // Return this if JSON has ALL required information. Else ask questions
+    data?: HealthData;
+    // Use this to ask questions and give pertinent responses
+    message?: string;
+    // Use this parts of the user request not translateed, off topic, etc
+    notTranslated?: string;
+}
+
+export interface HealthData {
+    medication?: Medication[];
+    condition?: Condition[];
+    other?: OtherHealthData[];
+}
+
+// Meds, pills etc.
+export interface Medication {
+    // Fix any spelling mistakes, especially phonetic spelling
+    name: string;
+    // E.g. 2 tablets, 1 cup. Required
+    dose: ApproxQuantity;
+    // E.g. twice a day. Required
+    frequency: ApproxQuantity;
+    // E.g. 50 mg. Required
+    strength: ApproxQuantity;
+}
+
+// Disease, Ailment, Injury, Sickness
+export interface Condition {
+    // Fix any spelling mistakes, especially phonetic spelling
+    name: string;
+    // When the condition started? Required
```

Comment:
Do we really need to specify "required" or "optional"? It should fall out of the declaration syntax, right?

```suggestion
    // When the condition started.
```